modules/phase_field/src/action/RigidBodyMultiKernelAction.C

```diff
@@ -50,29 +50,44 @@ RigidBodyMultiKernelAction::act()
     //
 
     std::string var_name = _var_name_base + Moose::stringify(op);
-    std::vector<VariableName> v;
+
+    //
+    // Create vector of coupled variables
+    //
+
+    std::vector<VariableName> arg;
     unsigned int ind = 0;
 
     if (isParamValid("c"))
     {
       VariableName c = getParam<VariableName>("c");
-      v.resize(_op_num);
+      arg.resize(_op_num);
 
       for (unsigned int j = 0; j < _op_num; ++j)
         if (j != op)
-          v[ind++] = _var_name_base + Moose::stringify(j);
+          arg[ind++] = _var_name_base + Moose::stringify(j);
 
-        v[ind++] = c;
+        arg[ind++] = c;
     }
     else
     {
-      v.resize(_op_num - 1);
+      arg.resize(_op_num - 1);
       for (unsigned int j = 0; j < _op_num; ++j)
         if (j != op)
-          v[ind++] = _var_name_base + Moose::stringify(j);
+          arg[ind++] = _var_name_base + Moose::stringify(j);
     }
 
     //
+    // Create vector of order parameters
+    //
+
+    std::vector<VariableName> v;
+    unsigned int i = 0;
+    v.resize(_op_num);
```

Comment:
No need to have a resize operation here. You can size the vector as you create it.